app/assets/javascripts/leaflet.map.js

```diff
@@ -11,6 +15,40 @@ L.extend(L.LatLngBounds.prototype, {
   }
 });
 
+if (OSM.MAPTILER_KEY) {
+  L.OpenMapTiles = L.MaplibreGL.extend({
```

Comment:
Arabic and Hebrew would be broken unless this code also [installs the right-to-left text plugin](https://maplibre.org/maplibre-gl-js-docs/example/mapbox-gl-rtl-text). (Note that it only supports these two languages, not other languages in the same scripts or other right-to-left scripts.)